tests/test_execution.py

```diff
@@ -4,6 +4,9 @@
 
 import substratest as sbt
 
+status_done = 'done'
+status_failed = 'failed'
```

Comment:
What about creating an TupleStatus type in the `substratest.assets` module?
It could be an enum.